lib/galaxy/model/__init__.py

```diff
@@ -1258,6 +1258,21 @@ def update_output_states(self):
         for statement in statements:
             sa_session.execute(statement, params)
 
+    def remappable(self):
+        """
+        Find if job is remappable when rerun
+        """
+        if self.state == self.states.ERROR:
+            try:
+                if [hda.dependent_jobs for hda in [jtod.dataset for jtod in self.output_datasets] if hda.dependent_jobs]:
+                    return True
```

Comment:
I think this is equivalent and more readable?
```suggestion
                for jtod in self.output_datasets:
                    if jtod.dataset.dependent_jobs:
                        return True
```